src/lib/index.ts

```diff
@@ -80,7 +80,9 @@ export class GoogleToken {
           .then(t => {
             callback(null, t);
           })
-          .catch(callback);
+          .catch(e => {
+            callback(e);
+          });
```

Comment:
Why is this change necessary?

Reply:
Brain left head.  Had it there for debugging, fixed.